src/test/regress/sql/citus_local_tables.sql

```diff
@@ -446,6 +447,12 @@ SELECT create_citus_local_table('referenced_table');
 
 ALTER TABLE referencing_table ADD CONSTRAINT fkey_cl_to_cl FOREIGN KEY (a) REFERENCES referenced_table(a);
 
+-- verify creating citus local table with extended statistics
+CREATE TABLE test_citus_local_table_with_stats(a int, b int);
+CREATE STATISTICS stx1 ON a, b FROM test_citus_local_table_with_stats;
+SELECT create_citus_local_table('test_citus_local_table_with_stats');
+SELECT stxname FROM pg_statistic_ext ORDER BY stxname;
+
```

Comment:
some more test ideas:

* alter/add .. statistics on an existing citus local table
* alter/add .. statistics on a citus local table from a different schema
* statistics with weird names / maybe in schemas with weird names ("Bad!SchemaName", BadStatName")